### PR TITLE
Working version compatible with pyblish-qml 0.2.6

### PR DESCRIPTION
**Pull request by tokejepsen**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -0,0 +1,9 @@
+pyblish-nuke Changelog
+=================
+
+This contains all major version changes between pyblish releases.
```
Comment (mottosso) [inline CHANGES]: Sorry to nitpick, but this should say "..between **pyblish-nuke** releases". :)

**Comment by mottosso** [comment]
Thanks, looks good to me!

**Comment by mottosso** [comment]
Just awaiting Travis

**Comment by tokejepsen** [comment]
merge?

**Comment by mottosso** [comment]
Sorry, I thought you mentioned something about it not working in the Gitter and was awaiting that.

Merging now!

**Pull request merged**